packages/ui-react/src/containers/AccountModal/forms/Checkout.tsx

```diff
@@ -42,7 +42,8 @@ const Checkout = () => {
       onSubmitPaypalPaymentSuccess: ({ redirectUrl }) => {
         window.location.href = redirectUrl;
       },
-      onSubmitStripePaymentSuccess: () => navigate(modalURLFromLocation(location, 'waiting-for-payment'), { replace: true }),
+      onSubmitStripePaymentSuccess: () =>
+        navigate(modalURLFromLocation(location, 'waiting-for-payment', { offerId: selectedOffer?.planOriginalId }), { replace: true }),
```

Comment:
This is not compatible with generic intergrations

Reply:
Good catch.